src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Input/Command/CommandValueSerializer.cs

```diff
@@ -117,7 +117,7 @@ public override object ConvertFromString(string value, IValueSerializerContext c
         {
             if (value != null)
             {
-                if (value != String.Empty)
+                if (value.Length != 0)
```

Comment:
I think this code can merge two if.

if (!string.IsNullOrEmpty(value))

Reply:
There is an `else` clause for the nested if.

```
if (value != null)
{
    if (value.Length != 0)
    {
        // ......
    }
    else
    {
        // ....
    }
}
```

I can't imagine how can they both be combined into one condition.

Reply:
you could do:

```cs
if (string.IsNullOrEmpty(value)) return null;

// inner logic here...

return base.ConvertFromString(value, context);
```

Reply:
@Nirmal4G, if it's null, you'll return null by your code, but the original code will `return base.ConvertFromString(value, context);`.
I'd keep it as-is for this PR. I believe there might be a simplification for this nested if, but can't get it well in my head.

Reply:
OK. I see now, the `ValueSerializer`'s `ConvertFromString` method throws null exception, if the value is null. So, we're hooking up with `base`'s error checking.

> Better way is to separate the exception helpers into separate static class and use it across the codebase.